pkg/network/message.go

```diff
@@ -184,6 +184,9 @@ func (m *Message) decodePayload(br *io.BinReader) error {
 		p = &payload.AddressList{}
 	case CMDBlock:
 		p = &core.Block{}
+	case CMDConsensus:
+		// Stubbed out for now.
```

Comment:
just in case: I think it's good to leave links to issues in comments so anyone who finds such comment won't go through the list of issues to search whether we have issue for that or not.